lib/pkgcloud/openstack/database/client/instances.js

```diff
@@ -320,3 +320,70 @@ exports.setVolumeSize = function setVolumeSize(instance, newSize, callback) {
     }), callback);
   });
 };
+
+// Enable the root user for the given database instance.
+// ### @instance {string | Object} The ID of the instance or a instance or Instance class (required)
+// ### @callback {Function} Function to continue, the call is cb(error, res)
+exports.enableRootUser = function(instance, callback) {
```

Comment:
Is there a matching `disableRootUser` on the API? We should probably support that if we're going to enable it.